packaging/os/dnf.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/python -tt
-# -*- coding: utf-8 -*-
-
-# Written by Cristian van Ee <cristian at cvee.org>
+# vim: set fileencoding=utf-8
+#
```

Comment:
That's emacs notation, I'm using vim's notation (`vim: set fileencoding=utf8`). Both notation works, although if it's standard to use the emacs notation I'm happy to change it.